packages/hop-node/src/theGraph/getTransferFromL1Completed.ts

```diff
@@ -0,0 +1,58 @@
+import makeRequest from './makeRequest'
+import { MaxInt32 } from 'src/constants'
+import { constants } from 'ethers'
+import { normalizeEntity } from './shared'
+
+export default async function getTransferFromL1Completed (
+  chain: string,
+  token: string,
+  startDate: number = 0,
+  endDate: number = MaxInt32,
+  lastId: string = constants.AddressZero
+) {
+  const query = `
+    query TransferFromL1Completed(${token ? '$token: String, ' : ''}$startDate: Int, $endDate: Int, $lastId: ID) {
```

Comment:
suggestion: since the `token` parameter for the function is required then `$token` here should always be set (no ternary)


```gql
    query TransferFromL1Completed($token: String, startDate: Int, $endDate: Int, $lastId: ID) {
```